clients/privacy-center/features/common/config.slice.ts

```diff
@@ -1,39 +1,33 @@
-import { createSelector, createSlice, PayloadAction } from "@reduxjs/toolkit";
-import { produce } from "immer";
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 
+import { useAppSelector } from "~/app/hooks";
 import type { RootState } from "~/app/store";
-import { config as initialConfig } from "~/constants";
+import { getDefaultConfig } from "~/constants";
 import { Consent, ConsentPreferences } from "~/types/api";
-import { ConfigConsentOption } from "~/types/config";
-
-type State = {
-  consent?: {
-    page: {
-      consentOptions?: ConfigConsentOption[];
-    }
-  };
-};
-const initialState: State = {};
+import { Config } from "~/types/config";
+
+// TODO: by statically loading the config here, we *guarantee* that the initial
+// state is never undefined. This is convenient for all the Typescript checking,
+// but is probably not wise - do we *really* want to ever show the "default"
+// prebuilt config?
+//
+// We should refactor this to initialize the slice when the store is initialized
+// (at runtime) and then set the initial state then. That would allow us to define the type as:
+// ```
+// type State = Config | undefined
+// ```
+//
+// ...and then update all the code to handle that.
+type State = Config
+const initialState: State = getDefaultConfig();
 
 export const configSlice = createSlice({
   name: "config",
   initialState,
   reducers: {
-    overrideConsentOptions(
```

Comment:
This action was only used by the Cypress test. A version of this is reasonable, but I'd prefer to make it configure the _entire_ config object and not just the consent options; more explicit